tests/queries/0_stateless/02534_default_granularity.reference

```diff
@@ -0,0 +1 @@
+CREATE TABLE default.users_02534\n(\n    `id` Int16,\n    `name` String,\n    INDEX bf_idx name TYPE minmax GRANULARITY 1\n)\nENGINE = MergeTree\nORDER BY id\nSETTINGS index_granularity = 8192
```

Comment:
Pls also adjust docs, e.g. [here](https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/mergetree/#table_engine-mergetree-data_skipping-indexes). But possibly also in other places.